tests/acceptance/suites/client-sdk-http-header.js

```diff
@@ -0,0 +1,51 @@
+import { PORT, UNIVERSAL_AUTOCOMPLETE_URL_REGEX, UNIVERSAL_SEARCH_URL_REGEX } from '../constants';
+import { SearchRequestLogger } from '../searchrequestlogger';
+import { VERTICAL_SEARCH_URL_REGEX, VERTICAL_AUTOCOMPLETE_URL_REGEX } from '../constants';
```

Comment:
nit: can we combine the imports from constants?

Reply:
yup!